### DISCUSSION TITLE
Pagination does not work in fresh install

### DISCUSSION THREAD

**Discussion by ghost** [datenstrom/community#561] [Report a bug] [answered]
Hello,
Fresh installation v0.8.16 (blog) without modification, seven test posts: pagination does not work.

Fresh installation v0.8.15 (blog), without modification, seven test posts: pagination works.

Update v0.8.15 to v0.8.16 : pagination works.

Sorry, I'm French, not computer scientist and I don't speak English (DeepL assists me) but Datenstrom Yellow is exactly what I need, in form and spirit (thanks).

**Comment by markseuffert** [comment, marked as answer]
Hello, I updated the pagination to the new API. Now a fresh installation should work, please try it out.

**Comment by ghost** [comment]
> Now a fresh installation should work, please try it out.

Exactly, the pagination now works perfectly after a fresh installation (v0.8.16). Thank you very much.
